MovieApp/SceneDelegate.swift

```diff
@@ -16,7 +16,12 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         // Use this method to optionally configure and attach the UIWindow `window` to the provided UIWindowScene `scene`.
         // If using a storyboard, the `window` property will automatically be initialized and attached to the scene.
         // This delegate does not imply the connecting scene or session are new (see `application:configurationForConnectingSceneSession` instead).
-        guard let _ = (scene as? UIWindowScene) else { return }
+        guard let windowscene = (scene as? UIWindowScene) else { return }
+        
+        window = UIWindow(windowScene: windowscene)
+        let vc = MovieDetailsViewController()
```

Comment:
Don't use this kind of naming, `vc`. State the full name, ie. `viewController`, or `movieDetailsListViewController`.